Assets/Scripts/Resources/ResourceSpawner.cs

```diff
@@ -64,16 +64,19 @@ void Start()
 
 	void OnDrawGizmos()
 	{
-		Transform trans = GetComponent<Transform>();
-		Gizmos.DrawIcon( trans.position + Vector3.up * 0.5f, "ResourceSpawner.png", true );
+		Transform transform = GetComponent<Transform>();
+
+		Gizmos.matrix = transform.localToWorldMatrix;
+
+		Gizmos.DrawIcon( transform.position + Vector3.up * 0.5f, "ResourceSpawner.png", true );
```

Comment:
Is this set to a weird position because Gizmos.matrix is set to transform.localToWorldMatrix